game/tests.py

```
@@ -2229,7 +2229,8 @@ def test_chess_game_draws_on_insufficient_material_cpp_mocked(self):
 class TimeControlIncrementTest(TestCase):
     """Test flexible time control and increment logic."""
 
-    def test_increment_applied_after_move(self):
+    @mock.patch('time.time', return_value=1000.0)
+    def test_increment_applied_after_move(self, mock_time):
```

Ruff (0.15.20)

[warning] Unused method argument: `mock_time`

(ARG002)